package.json

```diff
@@ -36,10 +36,12 @@
     "@types/react": "^18.3.3",
     "@types/react-dom": "^18.3.0",
     "axios": "^1.6.8",
+    "chokidar": "^3.6.0",
```

Comment:
why install `"chokidar": "^3.6.0",`

Reply:
sir, i had no idea about this. during setting up Project locally, I got an error while running `npm install`  as per the Vs Code terminal Suggrstion I run `npm audit fix`, I guess that time these packages get installed, Can you please guide me upont this what to do next..;

Reply:
so, now you can pull our main branch or may be deleted or uninstall this because this can be create some issues for us and based on good practice I can except this package in current time

Reply:
I had uninstalled these packages. Thanks for Helping 😄